README.md

```diff
@@ -37,9 +62,13 @@ Provide a custom User-Agent:
 KnowSheet> HTTP(GET("http://httpbin.org/get").UserAgent("KnowSheet-REPL/1.0.0")).body
 ```
 
-To quit REPL, hit `Ctrl+C` twice:
+Provide extra HTTP headers:
+```
+KnowSheet> HTTP(GET("http://httpbin.org/get?query=1", HTTPHeaders().Set("Custom", "Header"))).body
+KnowSheet> HTTP(POST("http://httpbin.org/post", "BODY", "text/plain", HTTPHeaders().Set("Custom", "Header"))).body
```

Comment:
We can use multi-line commands like in node's REPL shell, right? Why not break this line then?